hashes/contrib/test.sh

```diff
@@ -53,7 +61,10 @@ if [ "$DO_FEATURE_MATRIX" = true ]; then
 fi
 
 if [ "$DO_SCHEMARS_TESTS" = true ]; then
-    (cd extended_tests/schemars && cargo test)
+    (
+        cd extended_tests/schemars
+        cargo test
+    )
```

Comment:
This is OK because `set -e` gets transferred to subprocess, subprocess is used to clean up `cd` afterwards.

Reply:
I saw yesterday @apoelstra using `pushd`/`popd`, I never use those commands and had totally forgotten about them. Would it be easier to maintain if we use them? The subprocess and `cd` does make readers have to think "why does this need a subprocess" and if someone comes along and removes the subprocess we may not catch it in review then other `cargo` commands could become bogus - I convinced myself, will change this.

Reply:
Good idea.